Rakefile

```diff
@@ -36,4 +36,4 @@ task :check_binstubs do
   end
 end
 
-task default: [:spec, :make_bin_executable, :yard, :rubocop, :check_binstubs]
+task default: %i(spec make_bin_executable yard rubocop check_binstubs)
```

Comment:
pretty sure this changes in a newer version of rubocop, but that change is I believe breaking so we can change that back when we update it.